src/Native/Bootstrap/common.h

```diff
@@ -73,7 +73,13 @@ struct RawEEType
 	void*       m_pIndirectionModule;
 };
 
-struct ReversePInvokeFrame;
+class Thread;
+
+struct ReversePInvokeFrame
```

Comment:
Since this is only used from code that is under a `WASMTODO` it doesn't exactly feel right to move this to common.

Do we even need this right now? It's not like we can take a GC anyway and before we can take one, the codegen will need to learn to do this anyway.

Reply:
Generated this from codegen instead (we do need it -- the GC won't allocate if we haven't called reverse pinvoke).